cltk/corpus/odia/alphabet.py

```diff
@@ -1,16 +1,80 @@
 """Odia alphabet"""
 __author__ = 'Nishchith Shetty <inishchith[at]gmail[.]com>'
 
-VOWELS = [
-    'ଅ', 'ଆ', 'ଇ', 'ଈ', 'ଉ', 'ଊ', 'ଋ',
-    'ୠ', 'ଌ', 'ୡ', 'ଏ', 'ଐ', 'ଓ', 'ଔ']
+# Oriya Unicode Standard
+
+VOWELS = {
+    '0B05':'ଅ',
+    '0B06':'ଆ',
+    '0B07':'ଇ',
+    '0B08':'ଈ',
+    '0B09':'ଉ',
+    '0B0A':'ଊ',
+    '0B0B':'ଋ',
+    'N/A':'ୠ',
+    '0B0C':'ଌ',
+    'N/A':'ୡ',
+    '0B0F':'ଏ',
+    '0B10':'ଐ',
+    '0B13':'ଓ',
+    '0B14':'ଔ'
+    }
+
+STRUCTURED_CONSONANTS = {
```

Comment:
A list instead of a dictionary would be better , just like [here](https://github.com/cltk/cltk/blob/master/cltk/corpus/greek/alphabet.py) . And same goes for vowels . Still wait for confirmation from @kylepjohnson before making any changes 😄 .

Reply:
Actually, I thought accessing unicodes would be more discrete and also having in dictionary format would make it more easier to access.

Reply:
@inishchith Should I convert the dictionaries int the list form as given in the [link](https://github.com/cltk/cltk/blob/master/cltk/corpus/greek/alphabet.py) pointed by you or should we wait for @kylepjohnson ?